### PR TITLE
docs/podman-derivative-api.md: New file

### PR DESCRIPTION
**Pull request by cgwalters**
https://github.com/debarshiray/toolbox/ was planning to vendor
libpod which I disagreed with.  Let's document the advantages/disadvantages
as they exist today.

### CODE REVIEW

File: docs/podman-derivative-api.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Vendoring into a Go project
+---
+
+Advantages:
+
+ - Significant power and control
+ 
+Disadvantages:
+
+ - You are now on the hook for container runtime security updates (partially, `runc` is separate)
```
Comment (TomSweeneyRedHat) [inline docs/podman-derivative-api.md]: perhaps s/for instance `runc` and `crun` are separate/`runc` is separate/

**Comment by mheon** [comment]
@cgwalters Commit needs a signoff to make CI happy

File: docs/podman-derivative-api.md (hunk 1 above)
Comment (cgwalters) [reply docs/podman-derivative-api.md]: Fixed, thanks!

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by cgwalters** [comment]
Wow, the git-validation thing checking for trailing whitespace and and not making it obvious what the error is...rough.

**Comment by TomSweeneyRedHat** [comment]
Welcome to my document Git Hell @cgwalters

**Comment by TomSweeneyRedHat** [comment]
@mheon, happy green test buttons here.

**Pull request merged**